MyComicsManager.sln

```diff
@@ -31,7 +31,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Domain.UnitTests", "Domain.
 		{E8EC4431-1756-4298-B286-679FCD2D75C0} = {E8EC4431-1756-4298-B286-679FCD2D75C0}
 	EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Application.IntegrationTests", "Apllication.IntegrationTests\Application.IntegrationTests.csproj", "{CF4FD0CF-1A8E-4B53-B409-AAD2EA1C820C}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Application.IntegrationTests", "Apllication.IntegrationTests\Application.IntegrationTests.csproj", "{CF4FD0CF-1A8E-4B53-B409-AAD2EA1C820C}"

```

Comment:
**Suggestion:** There seems to be a typo in the project path for `Application.IntegrationTests`. It is written as `Apllication.IntegrationTests` instead of `Application.IntegrationTests`. [bug]
```suggestion
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Application.IntegrationTests", "Application.IntegrationTests\Application.IntegrationTests.csproj", "{CF4FD0CF-1A8E-4B53-B409-AAD2EA1C820C}"
```